app/views/general_reports/_report_rows.html.haml

```diff
@@ -2,11 +2,19 @@
   - @rows.each do |row|
     %tr
       - row.each_with_index do |col, i|
-        %td{:class => i == 0 ? '' : 'currency'}= i == 2 ? number_to_currency(col) : i == 3 ? number_to_percentage(col, :precision => 1) : col
+        %td{class: i == 0 ? '' : 'currency'}
+          - if i == 2
```

Comment:
I'd probably lean toward a `case`/`when` structure instead but it's not a showstopper. This is so much clearer than what was there.